tsconfig.base.json

```diff
@@ -27,8 +27,8 @@
       "@energinet-datahub/dh/admin/data-access-graphql": [
         "libs/dh/admin/data-access-graphql/src/index.ts"
       ],
-      "@energinet-datahub/dh/admin/feature-create-user-role": [
-        "libs/dh/admin/feature-create-user-role/src/index.ts"
+      "@energinet-datahub/dh/admin/feature-create-user-role-modal": [
```

Comment:
Consideration: Maybe not name the library after the layout but functionality/business logic, otherwise it will need a rename again if the UX changes back to a page or uses a drawer or what so ever.

Reply:
Good point. Will rename the library.